mediator/test/e2e-test.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -e
+BASEDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && cd .. && cd .. && pwd )"
+MEDIATORDIR="${BASEDIR}/mediator"
+DOCKERRDIR="${BASEDIR}/docker"
+
+
+export NODE_ENV=integration
+export NODE_TLS_REJECT_UNAUTHORIZED=0
```

Comment:
@samuelimoisili @lorerod, per [this comment ](https://stackoverflow.com/a/65021950/3541845), when changing this line to:

```
export NODE_TLS_REJECT_UNAUTHORIZED=1
```

in the development environment, solves the error that we were getting. 

However, now I get a different error related to the mediator installation:

<img width="673" alt="Screenshot 2023-05-10 at 10 59 15" src="https://github.com/medic/cht-interoperability/assets/24651706/21802500-4ee1-4138-bf33-a1287ab092a0">

Reply:
I will look into this today. Thanks for the feedback!